### ISSUE TITLE
KoverVerify fails on companion object

### ISSUE THREAD

**Issue by BulatInDrive** [Kotlin/kotlinx-kover#495]
**Describe the bug**
I have a class with a companion object. Both class and companion object have 1 function each. The validation rule is set to `minValue = 20`. The class function has test, but the companion object does not. When I run `koverVerify`, I get an error.

**Errors**
`Rule violated: lines covered percentage for class 'tat.mukhutdinov.moduleB.b1.HelperB1$Companion' is 0.000000, but expected minimum is 20`

**Expected behavior**
Verify should complete successfully since total class coverage is 66.7%

**Reproducer**
```
package tat.mukhutdinov.moduleB.b1

class HelperB1 {

    fun sort(a: Int, b: Int, c: Int): List<Int> =
        listOf(a, b, c).sorted()

    companion object {
        fun foo() {
            println("foo")
        }
    }
}
```
<img width="1557" alt="image" src="https://github.com/Kotlin/kotlinx-kover/assets/127848635/035cb1e0-5cae-4c7f-80a5-727305e2c260">

```
koverReport {
    defaults {
        verify {
            rule {
                entity = GroupingEntityType.CLASS

                filters {
                    includes {
                        classes("tat.mukhutdinov.moduleB.b1.HelperB1")
                    }

                }

                bound {
                    minValue = 20
                    metric = MetricType.LINE
                    aggregation = AggregationType.COVERED_PERCENTAGE
                }
            }
        }
    }
}
```

**Environment**
 - Kover Gradle Plugin version: 0.7.3
 - Gradle version: 7.3.1
 - Kotlin project type: Kotlin/Android
 - Coverage Toolset (if customized in build script): Kover

**Comment by shanshin** [comment]
Hi,
at the moment, Kover evaluates coverage by JVM classes, so all nested or companion classes have an individual percentage of coverage.

**Comment by shanshin** [comment]
Closed in favor of #498
